solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -2655,19 +2656,17 @@ public void initializeMetrics(
                 "solr_searcher_warmup_time", "Searcher warmup time (ms)", OtelUnit.MILLISECONDS),
             baseAttributes);
 
-    // fullSortCount
+    // Query sorting counts
     toClose.add(
         solrMetricsContext.observableLongCounter(
-            "solr_searcher_full_sort",
-            "Number of queries that required building a full DocList with sorting",
-            obs -> obs.record(fullSortCount.sum(), baseAttributes)));
-
-    // skipSortCount
-    toClose.add(
-        solrMetricsContext.observableLongCounter(
-            "solr_searcher_skip_sort",
-            "Number of queries where Solr skipped building a full sorted DocList and instead returned results directly from the DocSet",
-            obs -> obs.record(skipSortCount.sum(), baseAttributes)));
+            "solr_searcher_queries_sort",
```

Comment:
shouldn't the "sort" aspect entirely be an attribute?  The metric would count queries and the attributes would characterize multiple natures of the query -- sort is one and we may add more (there are many characteristics to a query beyond the sort!  docSet?  multiThreaded?  cursorMark?  hasPostFilter?  hasReRank? ... I could go on)

Reply:
I also support dropping this matter for now (no metric)